packages/superset-ui-chart-controls/src/components/MetricOption.tsx

```diff
@@ -33,20 +34,10 @@ const FlexRowContainer = styled.div`
 `;
 
 export interface MetricOptionProps {
-  metric: {
-    verbose_name: string;
-    metric_name: string;
-    label: string;
-    description: string;
-    warning_text: string;
-    expression: string;
-    is_certified?: boolean;
-    certified_by?: string | null;
-    certification_details?: string | null;
-  };
-  openInNewWindow: boolean;
-  showFormula: boolean;
-  showType: boolean;
+  metric: Metric;
```

Comment:
Just merged in that [other pr](https://github.com/apache-superset/superset-ui/pull/752/files) in case it affects this one.